### PR TITLE
BZ1852448 removing region from GCP UPI control plane doc

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1852448

@shellyyang1989, will you PTAL?

### CODE REVIEW

**Comment by shellyyang1989** [comment]
LGTM. Thanks.

File: modules/installation-creating-gcp-control-plane.adoc [hunk 1]
```diff
@@ -57,29 +57,27 @@ resources:
...
-<4> `control_subnet` is the `selfLink` URL to the control subnet.
-<5> `image` is the `selfLink` URL to the {op-system} image.
-<6> `machine_type` is the machine type of the instance, for example `n1-standard-4`.
-<7> `service_account_email` is the email address for the master service account created above.
-<8> `ignition` is the contents of the `master.ign` file.
+<2> `zones` are the zones to deploy the control plane instances into, for example `us-central1-a`, `us-central1-b`, and `us-central1-c`.
+<3> `control_subnet` is the `selfLink` URL to the control subnet.
+<4> `image` is the `selfLink` URL to the {op-system} image.
+<5> `machine_type` is the machine type of the instance, for example `n1-standard-4`.
+<6> `service_account_email` is the email address for the master service account created above.
```
Comment (codyhoag) [inline modules/installation-creating-gcp-control-plane.adoc:79]: >! above adj
Do not use to indicate a relative location in a document, as in “the above restrictions.” Use “previous” or
“preceding.”

I know that call-out wasn't added in this PR, but wanted to mention it.

**Review by codyhoag — COMMENTED** [review]
One small comment; all looks good otherwise!

File: modules/installation-creating-gcp-control-plane.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/installation-creating-gcp-control-plane.adoc:79]: Might as well fix them when they're spotted, though. Thanks!

**Pull request merged**